modules/cluster-logging-deploy-cli.adoc

```diff
@@ -0,0 +1,406 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// * assemblies/cluster-logging-deploying.adoc
+
+[id="cluster-logging-deploy-cli_{context}"]
+= Installing cluster logging using the CLI
+
+[role="_abstract"]
+You can use the `oc` CLI to install the Elasticsearch and Cluster Logging operators.
+
+.Prerequisites
+
+* Ensure that you have the necessary persistent storage for Elasticsearch. Note that each Elasticsearch node
+requires its own storage volume.
++
+Elasticsearch is a memory-intensive application. By default, the `oc` CLI installs three Elasticsearch nodes with memory requests and limits of 16 GB. This initial set of three {product-title} nodes might not have enough memory to run Elasticsearch within your cluster. If you experience memory issues that are related to Elasticsearch, add more Elasticsearch nodes to your cluster rather than increasing the memory on existing nodes.
+
+.Procedure
+
+To install the Elasticsearch Operator and Cluster Logging Operator using the CLI:
+
+. Create a Namespace for the Elasticsearch Operator.
```

Comment:
@RiRa12621 Can you please confirm on if the _customer_ should be creating this Namespace?

@jonquilwilliams I believe that Rick's comment was that customers should _not_ be creating this Namespace (since they don't have permissions to do so), and that there is already a process in place to create this Namespace for customres.

Reply:
@wgordon17 @RiRa12621 I had Slack-chatted Rick earlier and he said my changes are correct. Then I saw you question above, and so I Slack-chatted Rick again to see if I misunderstood, but no answer yet. @RiRa12621 Please advise whether this sentence "Create a Namespace for the Elasticsearch Operator." should be removed. Thanks.

Reply:
Ok, lines 23-90 have been removed since the cusotmer does not have to create the namespace. This should be correct now.